src/components/ReportActionItem/ReportActionItemImage.tsx

```diff
@@ -46,6 +48,9 @@ type ReportActionItemImageProps = {
 
     /** Whether there are other images displayed in the same parent container */
     isSingleImage?: boolean;
+
+    /** Whether the map view should not have border radius  */
+    shouldMapHaveNoBorderRadius?: boolean;
```

Comment:
Please remove the negation from the name of the boolean. `shouldMapHaveBorderRadius` and default it to true.